ams/server/src/test/java/com/netease/arctic/server/optimizing/TestMixedHiveOptimizing.java

```diff
@@ -136,7 +140,10 @@ private List<Record> readHiveTableData() throws TException, IOException {
   }
 
   private List<String> filesInLocation(String location) {
-    return arcticTable.io().list(location).stream().map(fileStatus -> fileStatus.getPath().toString())
-        .collect(Collectors.toList());
+    try (ArcticHadoopFileIO io = ((SupportHive) arcticTable).io()) {
```

Comment:
As far as I know, we don't need to close the IO after use because the current implementation is empty, and the table IO is shared, so it should not be closed after usage.